Web/starter_project_iconics/components/layout/Header.tsx

```diff
@@ -0,0 +1,118 @@
+import Image from 'next/image'
+import { useState } from 'react'
+import { MdOutlineClose, MdViewHeadline } from 'react-icons/md'
+import logo from '../../public/a2sv-logo.png'
+
+function classNames(...classes: string[]) {
+  return classes.filter(Boolean).join(' ')
+}
+
+const Header = () => {
+  const [navigation, setNavigation] = useState(() => [
+    { id: 0, name: 'Home', to: '/', current: true },
```

Comment:
1. idk why you need the id section as long as it is the same as the index
2. idk if tracking current this way is the best (i think current might be best tracked as a separate number - if you for instance want to set the current based on the url that is loaded, you can have a function that can do that on page load)
3. why do we have `/Success_story/success_stories` as the link? i don't understand why they have to be sub folders (also let us stay consistent with kebab-case for them too - not underscores and capitals)
4. i think there might be preexisting tabs that you can use that do this (check if they exist)